streaming/src/main/scala/org/apache/spark/streaming/scheduler/StreamingListenerBus.scala

```diff
@@ -17,17 +17,28 @@
 
 package org.apache.spark.streaming.scheduler
 
-import java.util.concurrent.atomic.AtomicBoolean
+import org.apache.spark.scheduler.{LiveListenerBus, SparkListener, SparkListenerEvent}
+import org.apache.spark.util.ListenerBus
 
-import org.apache.spark.Logging
-import org.apache.spark.util.AsynchronousListenerBus
+/**
+ * A Streaming listener bus to forward events in WrappedStreamingListenerEvent to StreamingListeners
```

Comment:
Add some docs to explain how this class works - why is it a listener, how does it forward, etc.